src/codeview.js

```diff
@@ -167,6 +167,8 @@ var Codeview = GObject.registerClass({
             buffer: this._buffer,
             showLineNumbers: true,
             visible: true,
+            tab_width: 4,
+            insert_spaces_instead_of_tabs: true,
```

Comment:
:+1:

Reply:
No reason to block the PR now, but for the future, remember to use camelCase for properties as in the rest of this property list.